src/FakeItEasy/TypeExtensions.cs

```diff
@@ -31,6 +32,15 @@ public static bool IsNullable(this Type type)
                 || (type.IsGenericType && type.GetGenericTypeDefinition() == typeof(Nullable<>));
         }
 
+        public static bool IsByRefLike(this Type type)
+        {
+#if LACKS_ISBYREFLIKE
+            return type.GetCustomAttributesData().Any(att => att.AttributeType.FullName == "System.Runtime.CompilerServices.IsByRefLikeAttribute");

```

Comment:
Well, that wasn't so bad after all. As I said in the other PR, I don't think we really _need_ to handle this case for netstandard2.0, but if it's no harder than that, I guess it doesn't hurt. We can always drop this method when we drop netstandard TFMs.

Reply:
Yeah, it ended up being simpler than I thought.